index.html

```diff
@@ -57,13 +64,181 @@ <h1>TechConnect <br> Web Development Workshop</h1>
 
 
 
-    <!-- <section class="About-TechConnect">
-        
-            <h2 class="About-h2">About-TechConnect </h2>
-            <p class="About-P">The "TechConnect: Web Development Workshop" website aims to provide a platform for aspiring web developers to learn and enhance their skills in the field of web development. The website will offer a range of resources, tutorials, and interactive activities to help participants grasp the fundamental concepts of web development and stay updated with the latest industry trends.</p>
+    <section class="container">
+
+        <div class="title">
+            <h2 class="About-h2">OUR-PROGRAM 
+                <hr class="section-title_line">
+            </h2>
+        </div>
+
+        <div class="box">
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development</p>
+                </div>
+
+            </div>
+
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development.</p>
+                </div>
+
+            </div>
+
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development.</p>
+                </div>
+
+            </div>
+
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development.</p>
+                </div>
+
+            </div>
+
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development</p>
+                </div>
+
+            </div>
 
```

Comment:
- [ ] `Required`
- [ ] Like mentioned above, the pages should look almost identical, this means typography, background.
- [ ] Please Make some adjustments 🥂 

`your template`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/f2e760aa-443e-42f3-9979-c555cbe3a62c)

`original template`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/413d8ded-a234-4d7f-bd99-8408a9289fc2)